client/app/components/dashboards/dashboard-grid.jsx

```diff
@@ -126,6 +167,8 @@ class DashboardGrid extends React.Component {
           margin={[cfg.margins, cfg.margins]}
           isDraggable={this.props.isEditing}
           isResizable={this.props.isEditing}
+          onResizeStart={this.autoHeightCtrl.stop}
```

Comment:
When widget is manually resized, interval is paused and then resumed to avoid "height fights" and also better perf.